packages/connectivity/src/scp-cf/destination/proxy-util.ts

```diff
@@ -29,6 +29,12 @@ export function proxyStrategy(destination: Destination): ProxyStrategy {
     );
     return ProxyStrategy.ON_PREMISE_PROXY;
   }
+
+  if (destination.proxyType === 'PrivateLink') {
+    logger.info('PrivateLink destination proxy settings will be used.');
```

Comment:
In Java SDK they also mention that this proxy type is not supported in local CI/CD enviroment. I would also mention this here.